tests/test_samples/no-doc_directive.cmake

```diff
@@ -0,0 +1,15 @@
+#[[[
+# This is an undocumented function
+#
+# :cminx: no-doc
+#]]
+function(undocumented_function)
+endfunction()
+
+#[[[
+# This is an undocumented macro
+#
+# :cminx: no-doc
+#]]
+macro(undocumented_macro)
+endmacro()
```

Comment:
Would be good to have another test case with both no-doc and regular documented functions and macros. Also, testing the directive on non-function/macro definitions would be good, as the current implementation should work on all supported documentable commands.

Additionally, the name of this test should be changed, as the `:cminx:` part is not a directive but rather a fake option to the implicit `py:function` directive that is generated.